src/components/App/App.jsx

```diff
@@ -65,12 +71,12 @@ class App extends React.Component {
 
   loadUser() {
     const github = new GitHub()
-    github.getCurrentUser().then(user => {
-      this.setState({ user })
-    }).catch(error => {
-      console.error('failed to load user', error)
-      GitHubAuth.deleteToken()
-    })
+    github.getCurrentUser()
+        .then(user => this.onUserLoad(user))
```

Comment:
Ok, I'll keep the . on the second line and align it with `.get`. I hate trailing dots because then if you need to edit it you have to edit two lines rather than one. I think it's rather neat to have your additions/deletions be one line rather than two where possible. The spacing is awkward since I think convention is to indent twice for newlines for chained methods? I like the align better.